protocols/v2/noise-sv2/src/signature_message.rs

```diff
@@ -70,12 +71,21 @@ impl SignatureNoiseMessage {
     //
     // If an authority public key is not provided, the function assumes that the signature
     // is already valid without further verification.
+    #[cfg(feature = "std")]
     pub fn verify(self, pk: &XOnlyPublicKey, authority_pk: &Option<XOnlyPublicKey>) -> bool {
+        let now = std::time::SystemTime::now()
+            .duration_since(std::time::UNIX_EPOCH)
+            .unwrap()
+            .as_secs() as u32;
+        self.verify_with_now(pk, authority_pk, now)
+    }
+    pub fn verify_with_now(
```

Comment:
making this method #[inline] here, will help to reduce the stack space.

Reply:
done

Reply:
with rust-doc